src/main/scala/sigmastate/eval/CostingDataContext.scala

```diff
@@ -43,7 +44,9 @@ class CostingBox(
     bytesWithoutRef: Col[Byte],
     propositionBytes: Col[Byte],
     registers: Col[AnyValue],
-    var isCost: Boolean) extends TestBox(id, value, bytes, bytesWithoutRef, propositionBytes, registers) {
+    var isCost: Boolean,
+    val ergoBox: ErgoBox
```

Comment:
@ergomorphic I've added an instance of `ErgoBox` to `CostingBox` rather than add missing properties. It might look a bit dirty, but the alternative is to add all `ErgoBox` properties to `CostingBox` (except `value`) I like less. 

This needed for pulling boxes through TreeBuilding.

Reply:
Removed as being too dirty. Created https://github.com/ScorexFoundation/sigmastate-interpreter/issues/325